at_file_share/bin/at_file_share.dart

```diff
@@ -0,0 +1,95 @@
+import 'package:args/args.dart';
+import 'package:at_client/at_client.dart';
+import 'package:at_file_share/src/file_send_params.dart';
+import 'package:at_file_share/src/service/file_receiver.dart';
+import 'package:at_file_share/src/service/file_sender.dart';
+import 'package:at_onboarding_cli/at_onboarding_cli.dart';
+
+const String version = '0.0.1';
+
+ArgParser buildParser() {
+  return ArgParser()
+    ..addFlag(
+      'help',
+      abbr: 'h',
+      negatable: false,
+      help: 'Print this usage information.',
+    )
+    ..addOption(
+      'mode',
+      abbr: 'm',
+      mandatory: true,
+      help: 'File sharing mode - send or receive',
+    )
+    ..addOption('atKeysFilePath',
+        abbr: 'k', mandatory: true, help: 'atKeys file of the current atsign')
+    ..addOption('sender',
+        abbr: 's', mandatory: true, help: 'atsign sending the file')
+    ..addOption('receiver',
+        abbr: 'r', mandatory: true, help: 'atsign receiving the shared file')
+    ..addOption('filePath',
+        abbr: 'f', mandatory: false, help: 'path of file in local to share')
+    ..addOption('downloadDir',
+        abbr: 'd', mandatory: false, help: 'download dir when receiving files')
+    ..addOption('bucketName',
+        abbr: 'b',
+        mandatory: false,
+        help: 'bucket name of storj to upload the file')
+    ..addOption('env',
+        abbr: 'e',
+        mandatory: false,
+        defaultsTo: 'root.atsign.org',
+        help: 'environment to share the file (dev/prod)');
+}
+
+void printUsage(ArgParser argParser) {
+  print('Usage: dart at_file_share.dart <flags> [arguments]');
+  print(argParser.usage);
+}
+
+void main(List<String> arguments) async {
```

Comment:
Can you change the sender and receiver to use the CLIBase class instead, please? See https://github.com/atsign-foundation/at_demos/blob/trunk/at_rpc_demo/bin/arithmetic_client.dart for example

Reply:
Thanks Gary. done